.gitignore

```diff
@@ -3,8 +3,5 @@
 # Ignore Vim swap files.
 .*.sw*
 # Ignore bazel directories
-/bazel-bin
-/bazel-genfiles
-/bazel-out
-/bazel-remote-apis
-/bazel-testlogs
+**/bazel-*
```

Comment:
(just double-checked to confirm) This ignores bazel-generated symlinks in repo subdirs, not just the toplevel, which is significant because there now exist example workspaces where one can `cd` to and run builds.

Clarified the comment above.

Reply:
I get that `**/bazel-*` would indeed ignore that. But why can't we just use `bazel-*`? So without a leading slash, without a leading anything.